04_Transaction/src/module/module.ts

```diff
@@ -182,4 +207,20 @@ export class TransactionModule extends AbstractModule {
         .then(messageConfirmation => this._logger.debug("Transaction response sent:", messageConfirmation));
     }
   }
+
+  @AsHandler(CallAction.MeterValues)
+  protected async _handleMeterValues(
+    message: IMessage<MeterValuesRequest>,
+    props?: HandlerProperties
+  ): Promise<void> {
+    this._logger.debug("MeterValues received:", message, props);
+
+    // TODO: Add meterValues to transactions
```

Comment:
@thanaParis Are we not at least storing the `meterValues` in a separate table?

Reply:
They would be inaccessible if we did, there's only data endpoints for fetching Transactions that have meter values not just meter values. MeterValues is deprecated in 2.0.1, only included for ease of upgrading from 1.6 systems. MeterValue data processing is on my fast-follow list after establishing core & advanced security.